src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/DateTimeMarshaler.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.InteropServices.JavaScript
+{
+    public static class DateTimeMarshaler
+    {
+        public static string FromJavaScriptPreFilter => "return BINDING._pre_filter_date(value)";
+        public static string ToJavaScriptPostFilter => "return new Date(value)";
```

Comment:
We could omit `return` and make it real expression, not statement.

Reply:
It used to omit return, but that makes it impossible to have multiple statements

Reply:
As you described elsewhere about `_pre_filter_date`, if there need be more logic, people should move it to JavaScript file and call it.

Reply:
I've decided not to make this change in this PR because it complicates writing tests and benchmarks. I'm open to fixing it in the follow-up PR that will be coming to do other todo list items.